addons/mail/models/discuss/discuss_channel.py

```diff
@@ -471,6 +471,9 @@ def _action_unfollow(self, partner=None, guest=None, post_leave_message=True):
             ],
         )
 
+    def get_displayed_name(self, member=None, author=None):
+        return (member.partner_id.name, author.name)
```

Comment:
I don't like the name and the API of this method. The name should be more specific (partner name in the context of current channel), and the API should just return the final name. Or as I commented elsewhere, the html link.
And member param is not worth it, if we have a member we should directly call the method on member instead (see my other comment about it).